autoload/pandoc/folding.vim

```diff
@@ -109,10 +109,13 @@ endfunction
 function! pandoc#folding#FoldExpr()
     " with multiple splits in the same buffer, the folding code can be called
     " way too many times too often, so it's best to disable it to keep good
-    " performance.
+    " performance. Only enable when using the built-in method of improving
+    " performance of folds.
+    if g:pandoc#folding#fastfolds = 1
     if count(map(range(1, winnr('$')), 'bufname(winbufnr(v:val))'), bufname("")) > 1
```

Comment:
```suggestion
        if count(map(range(1, winnr('$')), 'bufname(winbufnr(v:val))'), bufname("")) > 1
            return
        endif
```

Can we nest the indents? (Yes I have OCD.)

Reply:
...hold on not sure how to suggest a multiline change, I didn't mean to add the extra lines.

Reply:
I knew this was coming the moment I committed ;)